release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1370,26 +1309,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
-|Managing machines with the Cluster API for {azure-full}
-|Not Available
-|Not Available
-|Technology Preview
-
 |Managing machines with the Cluster API for {vmw-full}
 |Technology Preview
 |Technology Preview
 |Technology Preview
 
-|Defining a {vmw-short} failure domain for a control plane machine set
-|General Availability
-|General Availability
-|General Availability
-
-|Cloud controller manager for {alibaba}
-|Removed
-|Removed
-|Removed
-
```

Comment:
these lines should stay as they were merged

Reply:
:heavy_check_mark:

Reply:
The general rule is to remove entries that have 3 consecutive "Removed" or GA entries.